README.md

```diff
@@ -251,6 +253,37 @@ SitecoreImageBuilder\Invoke-Build `
 
 ```
 
+### License file volume mount configuration for 9.3 docker images
+
+If you've been using the license file mounted approach in 9.2 docker images, you can still continue using the same approach in 9.3 docker images. Follow these steps
+
+1. Place your Sitecore license file at `C:\license\license.xml`, or override location using the environment variable `LICENSE_PATH` like so: `$env:LICENSE_PATH="D:\my\sitecore\licenses"`
+1. Switch directory to `.\windows\tests\9.3.x\`  and then run any of the docker-compose files, for example an XM with: `docker-compose --file .\docker-compose.xm.yml up` after the docker compose file has been updated following the instruction below
+
+#### For cm and cd
+1. Add a new volume mount configuration using the environment variable `LICENSE_PATH` to a folder inside the container, for example `C:\license`
+1. Add a new environment variable `SITECORE_LICENSE_LOCATION` that point to the license file path inside the container, for example: `C:\license\license.xml`
+
+```
+cm:
+  environment:
+    SITECORE_LICENSE_LOCATION: C:\license\license.xml
+  volumes:
+    - ${LICENSE_PATH}:C:\license
+```
+
+#### For xconnect, xconnect-automationengine, xconnect-indexworker, xconnect-processingengine
+1. Add a new volume mount configuration using the environment variable `LICENSE_PATH` to a folder inside the container, for example `C:\license`
+1. Add a new environment variable `SITECORE_LICENSE_LOCATION` that point to the license file folder path inside the container, for example: `C:\license`
+
+```
+xconnect:
+  environment:
+    SITECORE_LICENSE_LOCATION: C:\license
```

Comment:
Why in some cases `SITECORE_LICENSE_LOCATION` is `C:\license` and in some other places it's `C:\license\license.xml`? That's confusing (unless i'm missing smth.).

Reply:
xconnect roles has the license file name hardcoded to be "license.xml", this has been reported to Sitecore :)